apps/website-25/src/components/courses/exercises/FreeTextResponse.stories.tsx

```diff
@@ -21,5 +21,13 @@ export const Default: Story = {
   args: {
     title: 'Understanding LLMs',
     description: 'Why is a language model\'s ability to predict \'the next word\' capable of producing complex behaviors like solving maths problems?',
+    onExerciseSubmit: () => {},
```

Comment:
nit: Use alert(JSON.log()) or console.log() so it's easy for people in storybook to see what the submit callback looks like?